concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -0,0 +1,901 @@
+                  - -c
+                  - -e
+                  - |
+                    ruby paas-cf/concourse/scripts/extract_terraform_state_to_yaml.rb \
+                      < vpc-tfstate/vpc.tfstate \
+                      > terraform-outputs/vpc.terraform-outputs.yml
+                    ruby paas-cf/concourse/scripts/extract_terraform_state_to_yaml.rb \
+                      < bosh-tfstate/bosh.tfstate \
+                      > terraform-outputs/bosh.terraform-outputs.yml
+
+      - task: render-manifest
+        config:
+          platform: linux
+          image: docker:///governmentpaas/spruce
+          inputs:
+            - name: paas-cf
+            - name: terraform-outputs
+            - name: generated-bosh-secrets
+            - name: generated-bosh-CA
+          outputs:
+            - name: bosh-manifest
+          params:
+            BOSH_MANIFEST_STUBS: |
+              ./generated-bosh-secrets/bosh-secrets.yml
+              ./generated-bosh-CA/bosh-ca-cert.yml
+              ./terraform-outputs/bosh.terraform-outputs.yml
+              ./terraform-outputs/vpc.terraform-outputs.yml
+          run:
+            path: sh
+            args:
+              - -e
+              - -c
+              - |
+                ./paas-cf/manifests/bosh-manifest/build_manifest.sh $BOSH_MANIFEST_STUBS > bosh-manifest/bosh-manifest.yml
+                /bin/cat bosh-manifest/bosh-manifest.yml
+        on_success:
+          put: bosh-manifest
+          params:
+            file: bosh-manifest/bosh-manifest.yml
+
+  - name: bosh-deploy
+    serial_groups: [bosh-deploy]
+    serial: true
+    plan:
+      - aggregate:
+          - get: pipeline-trigger
+            passed: ['generate-bosh-config']
+            trigger: true
+          - get: bosh-manifest
+            passed: ['generate-bosh-config']
+          - get: bosh-init-state
+          - get: ssh-private-key
+
+      - task: bosh-init-microbosh
+        config:
+          platform: linux
+          image: docker:///governmentpaas/bosh-init
+          inputs:
+            - name: bosh-manifest
+            - name: bosh-init-state
+            - name: ssh-private-key
+          outputs:
+            - name: updated-bosh-init-state
+          run:
+            path: sh
+            args:
+              - -e
+              - -c
+              - |
+                mkdir -p bosh-manifest/.ssh
+                cp ssh-private-key/id_rsa bosh-manifest/.ssh/id_rsa
+                chmod 400 bosh-manifest/.ssh/id_rsa
+                cp bosh-init-state/bosh-manifest-state.json bosh-manifest/bosh-manifest-state.json
+                bosh-init deploy bosh-manifest/bosh-manifest.yml
+                cp bosh-manifest/bosh-manifest-state.json updated-bosh-init-state/bosh-manifest-state.json
+        ensure:
+          put: bosh-init-state
+          params:
+            file: updated-bosh-init-state/bosh-manifest-state.json
+
+  - name: generate-cf-secrets
+    serial_groups: [cf-deploy]
```

Comment:
The name of serial group is different for BOSH and CF part. Does it mean that we can run both parts in parallel? I have an impression that this can lead to race conditions.